src/config/dev.env

```diff
@@ -0,0 +1,3 @@
+PORT = 6000
+DB_CONNECT = mongodb://localhost:27017/Heai-backend_DB
```

Comment:
create `.env.example` which should be pushed to GitHub. this file should be in the root directory. Add config folder to `gitignore`